### DISCUSSION TITLE
sending address via sharing command

### DISCUSSION THREAD

**Discussion by The-Darjeeling** [alandtse/tesla#196] [Q&A]
Hi, 

I'm trying to push addresses from HA to the car however im struggling with the Api command. 

looking at the API documnetation for [sharing](https://tesla-api.timdorr.com/vehicle/commands/sharing), ive come up with this but it doesnt seem to work

`service: tesla_custom.api
data:
  command: SHARE
  parameters:
    path_vars:
      vehicle_id: '{{state_attr(''binary_sensor.car_online_sensor'', ''id'') }}'
      type: share_ext_content_raw
    locale: en-gb
    timestamp_ms: '{{as_timestamp(now())}}'
    value:
      android.intent.extra.TEXT: "123 Charming Street, fakeland ,F43 3KE"
    wake_if_asleep: true`

I think its to do with the android intent bit as im not quite sure on the correct syntax

**Comment by alandtse** [comment]
Given the nature of this question, perhaps you should ask in the discussions for Tim's repo and link it here. They have much more traffic than us and I'm guessing someone else has figured out the format.

**Reply by Megabytemb** [reply]
As an FYI, i'm working on a rewite, and adding this via a service call is on the list of things to implement.

It looks like there's actually 2 calls, so i'll need to do some testing.
SEND_TO_VEHICLE and SEND_SC_TO_VEHICLE

https://tesla-api.timdorr.com/miscellaneous/endpoints
